styles.css

```diff
@@ -0,0 +1,95 @@
+*{
+    box-sizing: border-box;
+    margin:0;
+}
+body{
+    background-color: hsl(30, 38%, 92%);
```

Comment:
w zadaniu jak to, gdy mamy jakiś zestaw styli, które będą współdzielone w całej aplikacji warto rozważyć dodanie zmiennych, żeby w przypadku zmiany podmienić wartość koloru w jednym miejscu, a nie w każdym gdzie występuje:
https://www.w3schools.com/css/css3_variables.asp